h2o-algos/src/main/java/hex/util/DimensionReductionUtils.java

```diff
@@ -113,9 +113,9 @@ public static TwoDimTable createScoringHistoryTableDR(LinkedHashMap<String, Arra
      * 
      * @param dinfo
      * @param vEigenIn
-     * @return
+     * @return transformed eigenvectors
      */
-    public static double[][] transformEigenVectors(DataInfo dinfo, double[][] vEigenIn) {
+    public static double[][] getTransformedEigenvectors(DataInfo dinfo, double[][] vEigenIn) {
```

Comment:
Again, unnecessary to rename method transformEigenVectors to getTransformedEigenvectors.

Reply:
Hi again, @wendycwong !

This name in particular was tricky for me. Since I assumed (from the name) the method only transforms / does some changes without returning anything, I have introduced the bug from commmit [b45ba79330013b2435369301dae58d835bb9c30c](https://github.com/h2oai/h2o-3/pull/1025/commits/b45ba79330013b2435369301dae58d835bb9c30c).

Again, I believe it would be better to rename the method for better clarity in future. If somebody decides to use the method, one is sure there is a return value rather than plain `void` procedure. Hence, I will leave it to the new naming.

Thanks for your comment!